Automattic/wp-calypso
# Use domain property to determine whether domain privacy must be removed before udpate

#### Changes proposed in this Pull Request

We're adding a flag to determine whether privacy must be removed for a domain before updating the contact information in D28964-code. This PR will use that flag instead of explicitly checking the registrar to determine if we have to prevent contact updates when privacy is turned on.

#### Testing instructions

Visit the `contacts-privacy` page for a domains from different registries and make sure that when the `mustRemovePrivacyBeforeContactUpdate` flag is true for a domain, that the notice requiring privacy be turned off is shown.

Check the `domains` endpoint and use the redux tool to make sure that the flag is set properly in the sites/domains/items state tree for various domains. The flag should be `false` for domains that don't have the property and `true` for domains where the property is set to `true`.
